PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/console/SparkScalaLivyConsoleRunProfileState.kt

```diff
@@ -49,25 +54,25 @@ class SparkScalaLivyConsoleRunProfileState(private val consoleBuilder: SparkScal
 
     override fun execute(executor: Executor, runner: ProgramRunner<*>): ExecutionResult? {
         val console = consoleBuilder.console
-
-        val livySessionProcessHandler = SparkLivySessionProcessHandler(SparkLivySessionProcess(session))
+        val progressBarScheduler = IdeaSchedulers(consoleBuilder.project)
+        val livySessionProcess = SparkLivySessionProcess(progressBarScheduler, session)
+        val livySessionProcessHandler = SparkLivySessionProcessHandler(livySessionProcess)
 
         console.attachToProcess(livySessionProcessHandler)
+        logSubject.subscribe({ typedMessage -> console.print(typedMessage) }, { err ->
+            console.print("Livy interactive session is stop due to the error ${err.message}", LOG_ERROR_OUTPUT)
```

Comment:
wording: `Livy interactive session is stopped due to error`

Reply:
Fix it!